docs/_config.yml

```diff
@@ -1,7 +1,5 @@
-# Site settings
 title: instantsearch.js
 description: instantsearch.js is a library of widgets designed to help you build high performance instant search experiences with Algolia
-baseurl: "/instantsearch.js" # the subpath of your site, e.g. /blog/
```

Comment:
@vvo why removing the `baseurl`? Is that the reason why don't manage to get it work inside the subdir?